dlrover/python/elastic_agent/torch/training.py

```diff
@@ -177,15 +177,21 @@ class ElasticLaunchConfig(LaunchConfig):
     tee: Union[Std, Dict[int, Std]] = Std.NONE
     training_log_file: str = ""
     failure_node_errors: str = ""
+    step_rank: int = -1
```

Comment:
Use a class object here to unify step info. Or u can define a more common class to include these fields.